packages/interaction-context/src/useShowFocus.ts

```diff
@@ -0,0 +1,13 @@
+import { useContext } from 'react';
+import { UsingKeyboardContext } from './InteractionContext';
+
+export default function useShowFocus() {
+  const usingKeyboard = useContext(UsingKeyboardContext);
```

Comment:
So this is the weird part – it returns null if the context doesn't exist, and undefined when it does (since that's the initial value we set for the context in our call to `createContext`). We consider this below when setting the value the hook returns.